_episodes/04-legalities.md

```diff
@@ -29,27 +29,24 @@ keypoints:
 >
 >
 > - [Software Carpentry: Version Control with Git (full: 20m)](http://swcarpentry.github.io/git-novice/) --
->  Go through `Licensing` submodule.
+>  Go through the `Licensing` submodule.
 > - [Four aspects to make science open “by design” and not as an after-thought (full: 20m)](http://dx.doi.org/10.1186/s13742-015-0072-7) --
->  a brief correspondence which recapitulate four aspects (trademarks, ownership, licensing, and participant consent)
+>  a brief correspondence which recapitulates the aforementioned four aspects: trademarks, ownership, licensing, and participant consent
 >  in the scope of the scientific practice.
 {: .callout}
 
-## Anonimization tools
+## Anonymization tools
 
 > ## External materials
 >
-> - [Software Carpentry: Version Control with Git (full: 10m)](http://open-brain-consent.readthedocs.io/en/latest/annon_tools.html) --
->   Review anonimization tools available for various types of data in neuroimaing.
-> - [Four aspects to make science open “by design” and not as an after-thought (full: 20m)](http://dx.doi.org/10.1186/s13742-015-0072-7) --
->  a brief correspondence which recapitulate four aspects (trademarks, ownership, licensing, and participant consent)
->  in the scope of the scientific practice.
+> - [Latest anonymization tools for neuroimaging (full: 10m)](http://open-brain-consent.readthedocs.io/en/latest/annon_tools.html) --
```

Comment:
This link is invalid and does not show anonymization tools.

Reply:
I also removed the "four aspects" article as it was listed above. Happy to pop it back in if needed